dev-tools/packer/.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: required
```

Comment:
This file won't be used anymore. What's the plan for generating and publishing nightlies? I am guessing that this unified build will run nightly and publish to artifactory?